pulpcore/cli/rpm/repository.py

```diff
@@ -110,3 +111,41 @@ def sync(
         href=repository_href,
         body=body,
     )
+
+
+@repository.command()
+@name_option
+@href_option
+@click.option("--add-href", type=str, help="Comma-separated list of rpm content HREFs")
+@click.option("--remove-href", type=str, help="Comma-separated list of rpm content HREFs")
```

Comment:
We use json input to specify the content in the file case. And i'd really like to keep the interface consistent, so the user does not need to learn a new way with every plugin.